Includes/Rosetta/Conditions/SelfCondition.hpp

```diff
@@ -87,6 +87,11 @@ class SelfCondition
     //! \return Generated SelfCondition for intended purpose.
     static SelfCondition IsTagValue(GameTag tag, int value,
                                     RelaSign relaSign = RelaSign::EQ);
+   
+    //! SelfCondition wrapper for checking name of entity.
+    //! \param name The card name to check condition.
+    //! \param isEqual Decide whether to check for EQ or NE.
+    static SelfCondition IsName(std::string name, bool isEqual = true);
```

Comment:
1. I'd like you to leave the comments in more detail.
2. Please change the type of argument `name` to `const std::string&`.

Reply:
Fixed.